packages/core/src/providers/Provider.tsx

```diff
@@ -63,7 +64,16 @@ export const HvProvider = ({
   const themes = parseThemes(baseTheme, name, customizations);
 
   return (
-    <>
+    <CacheProvider
+      /**
+       * Moves UI Kit styles to the top of the <head> so they're loaded first.
+       * This enables users to override the UI Kit styles if necessary.
+       */
+      value={createCache({
+        key: "css",
```

Comment:
Maybe something more unique to UI Kit? To avoid conflicts it the app uses its CacheProvider.

Reply:
Yes, you are right 🙏 I changed it to `hv-uikit-css`.